kernel/enter.c

```diff
@@ -757,11 +757,15 @@ int myst_enter_kernel(myst_kernel_args_t* args)
     __myst_kernel_args = *args;
     args = &__myst_kernel_args;
 
-    /* set the syslog level, depending on whether in TEE debug mode */
-    if (args->tee_debug_mode)
-        args->syslog_level = LOG_DEBUG;
-    else
-        args->syslog_level = LOG_NOTICE;
+    /* If no syslog level config was specified set defaults depending on whether
+     * in TEE debug mode */
+    if (args->syslog_level == -1)
```

Comment:
I had trouble verifying that args->syslog_level is initialized with `-1` in the command line case (it is in the JSON case).

Reply:
Command line case is handled by `tools/myst/host/utils.c:get_syslog_level_opts`. As 0/LOG_EMERG is an acceptable value for syslog_level, had to choose -1 for denoting no config.

Reply:
Oh good. Resolving conversation.